xep-0353.xml

```diff
@@ -262,21 +234,179 @@
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- ...some time passes... -->
+
+<!-- new session (device switch) -->
+<message from='juliet@capulet.example/tablet'
+         to='romeo@montague.example'
+         type='chat'>
+  <propose xmlns='urn:xmpp:jingle-message:1' id='x64sjjvkla37baka'>
+    <description xmlns='urn:xmpp:jingle:apps:rtp:1' media='audio'/>
+  </propose>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- Romeo accepts the call because it is assumed to be a continuation of the old session with id 'a73sjjvkla37jfea'... -->
+<message from='romeo@montague.example/orchard'
+         to='juliet@capulet.example'
+         type='chat'>
+  <accept xmlns='urn:xmpp:jingle-message:1' id='x64sjjvkla37baka'/>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- ...and finishes the old session with id 'a73sjjvkla37jfea' directly afterwards using reason <expired/> -->
+<!-- and including a <migrated/> element pointing to the new session id 'x64sjjvkla37baka' -->
+<message from='romeo@montague.example/orchard'
+         to='juliet@capulet.example'
+         type='chat'>
+  <finish xmlns='urn:xmpp:jingle-message:1' id='a73sjjvkla37jfea'>
+    <reason xmlns="urn:xmpp:jingle:1">
+      <expired/>
+      <text>Session migrated</text>
+    </reason>
+    <migrated to='x64sjjvkla37baka'/>
+  </finish>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- ...some more time passes... -->
+
+<!-- the new session is termianted normally by Romeo... -->
+<message from='romeo@montague.example/orchard'
+         to='juliet@capulet.example'
+         type='chat'>
+  <finish xmlns='urn:xmpp:jingle-message:1' id='x64sjjvkla37baka'/>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+<!-- and Juliet... -->
+<message from='juliet@capulet.example/tablet'
+         to='romeo@montague.example'
+         type='chat'>
+  <finish xmlns='urn:xmpp:jingle-message:1' id='x64sjjvkla37baka'/>
+  <store xmlns="urn:xmpp:hints"/>
+</message>
+
+]]></example>
+  </section2>
 </section1>
-<section1 topic='Acknowledgements' anchor='acks'>
-  <p>Thanks to Lance Stout for his feedback.</p>
+<section1 topic='Business Rules' anchor="business-rules">
+  <p>Participants MUST use &xep0280; and &xep0313; to make sure all devices of initiator and responder receive all messages exchanged by this protocol.
+  Without &xep0280; implementations would need to send copies of outgoing messages to their own bare jid, to inform their own devices about an event (like it was done with the &lt;accept/&gt; message in the old urn:xmpp:jingle:jingle-message:0 specification).</p>
+  <p>In a &xep0313; (or &xep0198;) catchup scenario client developers MAY choose to not show an "incoming call" UI upon receiving a &lt;propose/&gt; message because they could receive another message for the same jingle id later in the catchup process invalidating the &lt;propose/&gt; received before. Showing the "incoming call" UI as soon as receiving an &lt;accept/&gt; might comprise bad UX.</p>
```

Comment:
```suggestion
  <p>In a &xep0313; (or &xep0198;) catchup scenario client developers MAY choose to not show an "incoming call" UI upon receiving a &lt;propose/&gt; message because they could receive another message for the same Jingle session id later in the catchup process invalidating the &lt;propose/&gt; received before. Showing the "incoming call" UI as soon as receiving an &lt;accept/&gt; might comprise bad UX.</p>
```